liquid_node/commands.py

```diff
@@ -208,7 +214,8 @@ def start(job, hcl):
             job_checks[service] = checks
         return job_checks
 
-    jobs = [(job.name, get_job(job.template)) for job in config.jobs]
+    config_jobs = config.jobs if scope != 'hoover' else config.hoover_jobs
```

Comment:
This is exactly the kind of `if` that will make the code unmaintainable in no time. If you only want to deploy hoover, then @gabriel-v 's idea in https://github.com/liquidinvestigations/node/pull/125#pullrequestreview-255718556 makes more sense.